ddtrace/internal/schema/span_attribute_schema.py

```diff
@@ -0,0 +1,23 @@
+SPAN_ATTRIBUTE_SCHEMA_VERSIONS = dict()
+
+_DEFAULT_SPAN_SERVICE_NAMES = {"v0": None, "v1": "unnamed-python-service"}
+
+
+def register_schema_version(version):
+    def _wrap(func):
+        assert version not in SPAN_ATTRIBUTE_SCHEMA_VERSIONS, "duplicate version entry for {}".format(version)
```

Comment:
Trying to think about this, under what circumstance could we end up with a duplicate version entry? Why do we care?

Reply:
This may be better as a test; the idea is to prevent someone copy+pasting for a new version and forgetting to update the tag